### ISSUE TITLE
How can I customize colors for distinct groups based on my specific preferences?

### ISSUE THREAD

**Issue by GooLey1025** [vgteam/sequenceTubeMap#463]
![图片](https://github.com/user-attachments/assets/7306dde3-4b9e-45e0-9b85-f43a4ba2ac44)

I have seen visualization for my vg.xg file.  But now I want to choose the distinct color for me.
It seems color legend can not be customized.
![图片](https://github.com/user-attachments/assets/16bc26f5-b548-482f-9d05-6ded4aa8d6ec)

**Comment by adamnovak** [comment]
Yeah, we don't currently have granular color controls to allow coloring each path uniquely. We have some internal support, with the primary and alternate colors, for coloring a backbone reference path differently from other paths, but we don't let you color a specific path with a custom color.

It does sound like a good feature to add, though.
